packages/cw4/src/hook.rs

```diff
@@ -10,17 +10,13 @@ use cosmwasm_std::{to_binary, Binary, CosmosMsg, HumanAddr, StdResult, WasmMsg};
 /// old = Some, new = None -> Delete
 #[derive(Serialize, Deserialize, Clone, PartialEq, JsonSchema, Debug)]
 pub struct MemberDiff {
-    pub key: HumanAddr,
+    pub key: Addr,
     pub old: Option<u64>,
     pub new: Option<u64>,
 }
 
 impl MemberDiff {
-    pub fn new<T: Into<HumanAddr>>(
-        addr: T,
-        old_weight: Option<u64>,
-        new_weight: Option<u64>,
-    ) -> Self {
+    pub fn new<T: Into<Addr>>(addr: T, old_weight: Option<u64>, new_weight: Option<u64>) -> Self {
```

Comment:
Is there anything that really implements `Into<Addr>`? I think we can just use `Addr` here (or does this allow `&addr` as well as `addr` as args?)